### DISCUSSION TITLE
Any plan to include resteasy fix for CVE-2023-0482

### DISCUSSION THREAD

**Discussion by magixyu** [keycloak/keycloak#19265] [Q&A] [answered]
Vulnerability Details : 

CVE-2023-0482In RESTEasy the insecure File.createTempFile() is used in the DataSourceProvider, FileProvider and Mime4JWorkaround classes which creates temp files with insecure permissions that could be read by a local user.

**Comment by abstractj** [comment, marked as answer]
@magixyu yes, CVEs fixes are planned by milestones and will be available in the upcoming releases.  For more information, please see our list of issues: https://github.com/keycloak/keycloak. 

Our team recently updated most of its dependency and this should be reflected by scanning the nightly images, and they should be part of upcoming releases:
```
trivy image --ignore-unfixed --timeout 25m quay.io/keycloak/keycloak:nightly
```
